packages/mural-picker/src/components/mural-create/index.tsx

```diff
@@ -16,6 +16,7 @@ import { getCommonTrackingProperties } from '../../common/tracking-properties';
 import { CardListSection } from '../card-list/card-list-section';
 import { ErrorHandler } from '../types';
 
+import { getApiError } from '@muraldevkit/mural-integrations-mural-client';
```

Comment:
Can you merge it with the import at line 4 ?

Reply:
good idea!